packages/ingest/aws.js

```diff
@@ -48,340 +13,82 @@ function getEndpoint(local = false, port = 8000) {
-    const params = {
-      Ids: [id],
-      Rule: rule
-    };
-
-    return cwevents.removeTargets(params).promise();
+  static deleteTarget(id, rule) {
+    deprecate('@cumulus/ingest/aws/Events.deleteTarget', '1.17.0', '@cumulus/aws-client/CloudwatchEvents.deleteTarget');
+    return CloudwatchEvents.deleteTarget(id, rule);
   }
 
-  static async putTarget(rule, id, arn, input) {
-    const cwevents = new AWS.CloudWatchEvents();
-
-    const params = {
-      Rule: rule,
-      Targets: [ /* required */
-        {
-          Arn: arn,
-          Id: id,
-          Input: input
-        }
-      ]
-    };
-
-    return cwevents.putTargets(params).promise();
+  static putTarget(rule, id, arn, input) {
+    deprecate('@cumulus/ingest/aws/Events.putTarget', '1.17.0', '@cumulus/aws-client/CloudwatchEvents.putTarget');
+    return CloudwatchEvents.putTarget(rule, id, arn, input);
   }
 }
 
 class SQS {
-  static async getUrl(name) {
-    const queue = sqs();
-    const u = await queue.getQueueUrl({ QueueName: name }).promise();
-    return u.QueueUrl;
-  }
-
-  static async deleteQueue(queueUrl) {
-    const queue = sqs();
-    const params = {
-      QueueUrl: queueUrl
-    };
-
-    return queue.deleteQueue(params).promise();
-  }
-
-  static async receiveMessage(queueUrl, numOfMessages = 1, timeout = 30) {
-    const queue = sqs();
-    const params = {
-      QueueUrl: queueUrl,
-      AttributeNames: ['All'],
-      VisibilityTimeout: timeout,
-      MaxNumberOfMessages: numOfMessages
-    };
-
-    const messages = await queue.receiveMessage(params).promise();
-
-    // convert body from string to js object
-    if (Object.prototype.hasOwnProperty.call(messages, 'Messages')) {
-      messages.Messages.forEach((mes) => {
-        mes.Body = JSON.parse(mes.Body); // eslint-disable-line no-param-reassign
-      });
-
-      return messages.Messages;
-    }
-    return [];
-  }
-
-  static async sendMessage(queueUrl, message) {
-    let messageBody;
-    if (isString(message)) {
-      messageBody = message;
-    } else if (isObject(message)) {
-      messageBody = JSON.stringify(message);
-    } else {
-      throw new TypeError('body type is not accepted');
-    }
-
-    const params = {
-      MessageBody: messageBody,
-      QueueUrl: queueUrl
-    };
-
-    const queue = sqs();
-    return queue.sendMessage(params).promise();
-  }
-
-  static async deleteMessage(queueUrl, receiptHandle) {
-    const queue = sqs();
-    const params = {
-      QueueUrl: queueUrl,
-      ReceiptHandle: receiptHandle
-    };
-
-    return queue.deleteMessage(params).promise();
-  }
-
-  static async attributes(name) {
-    const u = await this.getUrl(name);
-    const queue = sqs();
-    const params = {
-      AttributeNames: ['All'],
-      QueueUrl: u
-    };
-
-    const attr = await queue.getQueueAttributes(params).promise();
-    attr.Attributes.name = name;
-    return attr.Attributes;
-  }
-}
-
-
-class ECS {
```

Comment:
This whole class wasn't used anywhere that I could see, so I removed it. Thoughts?

Reply:
+1 for removing it